app/src/main/res/layout/activity_login.xml

```diff
@@ -5,8 +5,8 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="@color/purple_500"
-    android:backgroundTint="@color/purple_500"
+    android:background="@color/primaryLightColor"
+    android:backgroundTint="@color/primaryLightColor"
```

Comment:
I seems to me that these lines are redundant now that we have a theme defined. I think android should automatically pick the theme colors for this stuff, but including it anyway shouldn't hurt anything.